lib/extends/commands/decidim/budgets/admin/import_proposals_to_budgets_extends.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module ImportProposalsToBudgetsExtends
+  extend ActiveSupport::Concern
+
+  included do
+    private
+
+    def selected_scope_id
+      params.dig("proposals_import", "scope_id")&.to_i
+    end
+
+    def proposals
+      if selected_scope_id.present?
```

Comment:
```suggestion
      return all_proposals if selected_scope_id.blank?
      
      children = Decidim::Scope.find(selected_scope_id).children
      children.present? ? all_proposals.where(decidim_scope_id: children.ids.push(selected_scope_id)) : all_proposals.where(decidim_scope_id: selected_scope_id)
```

Prefer to use Guard clause rather than if else condition to be more readable